apps/web/src/lib/mock/organizations.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+interface OrgMockData {
+  id: string;
+  name: string;
+  ownerId: string;
+}
+
+const organizationsMock: OrgMockData[] = [
+  {
+    id: "7b74f4d5-ef62-4c55-91fc-73eeb6a2f700",
+    name: "MrBeast",
+    ownerId: "1fb6b3d1-0bc7-4a7a-b7f3-79ab7e9d5cb9",
+  },
+  {
+    id: "a1742b12-40a0-48bb-8c42-02d1680c5d0a",
+    name: "spryciarze.pl",
+    ownerId: "e6d6bc24-6f4d-4f40-b7f1-ded2f7a72ac8",
+  },
+];
```

Comment:
If you delete this `interface` add `as const` after closing bracket of this array.

```suggestion
const organizationsMock = [
  {
    id: "7b74f4d5-ef62-4c55-91fc-73eeb6a2f700",
    name: "MrBeast",
    ownerId: "1fb6b3d1-0bc7-4a7a-b7f3-79ab7e9d5cb9",
  },
  {
    id: "a1742b12-40a0-48bb-8c42-02d1680c5d0a",
    name: "spryciarze.pl",
    ownerId: "e6d6bc24-6f4d-4f40-b7f1-ded2f7a72ac8",
  },
] as const;
```